controllers/groupsController.js

```diff
@@ -0,0 +1,22 @@
+const groupModel = require("../models/groupModel");
+const { isValidString, isEmptyString } = require("../utils/validation");
+
+const list = async (req, res) => {
+  if (!isValidString(req.params.uid) || isEmptyString(req.params.uid)) {
+    return res.status(400).send({ message: "[InvalidUid] Error occured" });
+  }
+
+  try {
+    const { uid } = req.params;
+    const groupListResult = await groupModel
+      .find({ ownerUid: uid })
+      .populate("keywordIdList", "keyword");
+    res.status(200).json({ groupLength: groupListResult.length, groupListResult });
```

Comment:
```suggestion
    res.status(200).json({ groupListLength: groupListResult.length, groupListResult });
```
`groupLength`는 그룹 내 데이터의 길이를 의미한다고 생각할 수 있을 것 같아요. 
그래서 `groupLength` 보단 `groupListLength` 로 명명하는게 더 직관적일 것 같은데, `groupLength`로 변수명을 지으신 이유가 있을까요?!
cc. @SsongQ-92

Reply:
저도 `groupListLength`가 좋아보이네요. `groupListResult`와의 통일성을 고려해볼 수도 있고, `groupLength`는 그룹 하나의 길이를 의미하는 것 같기도 하네요

Reply:
아 그렇네요! `groupListLength`로 수정하면 좋겠습니다. 반영해둘게요 :)